src/components/List/__snapshots__/List.test.tsx.snap

```diff
@@ -14,14 +14,16 @@ exports[`List List is horizontal 1`] = `
       </h2>
     </div>
     <ul
-      class="list-container"
-      role=""
+      class="list-container my-list-class"
+      role="list"
     >
       <li
         class="list-item my-list-item-class"
         style="padding: 8px 16px;"
       >
-        <div>
+        <div
+          id="undefined"
```

Comment:
These may be ignored as it's a jest snap and I manually pass the id in the render.